### ISSUE TITLE
Missing issue template

### ISSUE THREAD

**Issue by rcapile** [muxinc/mux-php#38]
on the README.md you mention

> If you run into problems, please raise a GitHub issue, filling in the issue template. We'll take a look as soon as possible.

but I could not find the template

**Comment by eropple** [comment]
Good catch. We've been getting ready to standardize this stuff, but haven't gotten there yet! For now I'll remove the issue template line, and I'll leave this issue open until we get there.
